### PR TITLE
Fix: FQT otc order encoding logic

### PR DESCRIPTION
**Pull request by dextracker**
## Description

swap rfq and otc in fillQuoteTransformerDataEncoder to match our enum
## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

**Review by abls — CHANGES_REQUESTED** [review]
I don't believe this will fix it. You have to remove the `otcOrders` field completely, it isn't present on-chain at all.

IMO would be best to just revert [this commit](https://github.com/0xProject/protocol/commit/036a2632679e3c3167162f47083d906fe305bbaa).

File: packages/protocol-utils/src/transformer_utils.ts [hunk 1]
```diff
@@ -87,6 +82,11 @@ export const fillQuoteTransformerDataEncoder = AbiEncoder.create([
             { name: 'fillSequence', type: 'uint8[]' },
             { name: 'fillAmount', type: 'uint256' },
             { name: 'refundReceiver', type: 'address' },
+            {
```
Comment (kyu-c) [inline packages/protocol-utils/src/transformer_utils.ts:85]: shouldn't there be an accompanying change in FQT (solidity code)?

Comment (dextracker) [reply packages/protocol-utils/src/transformer_utils.ts:85]: yup forgot to add from /protocol was in protocol-utils

**Review by kyu-c — APPROVED** [review]
LGTM

**Pull request merged**